packages/entity-browser/src/modules/entityBrowser/reducer.js

```diff
@@ -6,9 +6,28 @@ const showEntityDetail = (state, {payload}) => ({
   showDetailEntityId: payload.entityId
 })
 
+const simpleSearchFieldsToArray = simpleSearchFields => {
```

Comment:
const simpleSearchFieldsToArray = simpleSearchFields => (
  simpleSearchFields.split(',')
     .filter(s => s)
     .map(s => .trim())
 )

should do the job